integration/aggregate_documents_test.go

```diff
@@ -27,20 +29,22 @@ func TestAggregateGroupErrors(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct {
-		err        mongo.CommandError
-		altMessage string
-		skip       string
-		pipeline   bson.A
+		pipeline bson.A // required, aggregation pipeline stages
+
```

Comment:
I added a space between custom fields and err, altMessage, skip because I want them in the same order. Interestingly linter is quite happy if it has a space, normally I get `vet` or something complaining